### PR TITLE
Merge `Repository.pyi` back to source

### PR DESCRIPTION
**Pull request by trim21**
this file is too huge(4271 lines), so I didn't change code unless it will fail mypy.

Also merges overloads of `create_pull`.

### CODE REVIEW

**Comment by EnricoMi** [comment]
LGTM! Please go ahead merging the `create_pull` methods.

**Comment by trim21** [comment]
@EnricoMi done

File: tests/Repository.py [hunk 1]
```diff
@@ -1296,11 +1296,11 @@ def testGetSubscribers(self):
...
-            "Pull request created by PyGithub",
-            "Body of the pull request",
-            "topic/RewriteWithGeneratedCode",
-            "BeaverSoftware:master",
-            True,
+            title="Pull request created by PyGithub",
+            body="Body of the pull request",
+            base="topic/RewriteWithGeneratedCode",
+            head="BeaverSoftware:master",
+            maintainer_can_modify=True,
```
Comment (EnricoMi) [inline tests/Repository.py:1303]: with
```suggestion
            draft=False,
            maintainer_can_modify=True,
```
there would be no need to change the replay data, and we would test that attribute

Comment (trim21) [reply tests/Repository.py:1303]: good idea

**Pull request merged**